### DISCUSSION TITLE
Enclosure value after MIDI event

### DISCUSSION THREAD

**Discussion by willeke1234** [GrandOrgue/grandorgue#1182] [Help]
When I send a Bx Controller event to an enclosure, the new value of the enclosure is one less than the sent value.
Example: an enclosure has the same send and receive Bx Controller MIDI event. Lower limit/Off value is 0, Upper limit/On value is 127. When I send value 50, I get value 49 back.
Is this a bug or a feature?

**Comment by jefferycrowley** [comment]
You have to remember that in computers counting begins with [0].  I discovered this because I have a numeric read-out for the crescendo pedal status.    I have to set it to upper limit 33 taking in account of the [0] as a digit.  It now reads 0-32 as it should.

**Reply by midimusic** [reply]
@jefferycrowley 

I suspect you are still wrong, your upper limit ought be 32 and it should read 0-31. The same thing with midi channels they are actually 0-15 but most users see them as 1-16.

csw900
